recipes/azureml/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "azureml" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/azureml-{{ version }}.zip
+  sha256: 1b48afa1306d1f32133748585a3e5a6793557780105f4e51c2a978070dd432f1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools >=61.0
+    - setuptools-scm
+  run:
+    - python >=3.6
+    - requests
+    - pandas
```

I see python-dateutil in the metadata too:

```suggestion
    - requests
    - pandas
    - python-dateutil
```

---

done, but, where did you find that python-dateutil was also a dependency?